x/project/handler.go

```diff
@@ -194,9 +228,54 @@ func getProjectAccountMap(ctx sdk.Context, k Keeper, projectDid ixo.Did) map[str
 	return k.GetAccountMap(ctx, projectDid)
 }
 
-func addAccountToAccountProjectAccounts(ctx sdk.Context, k Keeper, projectDid ixo.Did, accountDid ixo.Did) auth.Account {
-	acc := k.CreateNewAccount(ctx, projectDid, accountDid)
-	k.AddAccountToAccountProjectAccounts(ctx, projectDid, accountDid, acc)
+func getAccountInAccountProjectAccounts(ctx sdk.Context, k Keeper, projectDid ixo.Did, accountID string) sdk.AccAddress {
```

Comment:
Should this not be getAccountInProjectAccounts?

Reply:
Yes